pkg/kubectl/cmd/logs/logs.go

```diff
@@ -64,6 +64,10 @@ var (
 		# Begin streaming the logs from all containers in pods defined by label app=nginx
 		kubectl logs -f -lapp=nginx --all-containers=true
 
+		# Begin streaming the logs from all containers in pods defined by label app=nginx.
+		# Each line will be prefixed with the log source (pod name and container name)
+		kubectl logs -f -lapp=nginx --all-containers=true --prefix
```

Comment:
I'd drop this example, we don't need to have examples for every possible combination. You could add a single line of comment to the previous about `--prefix` possibility. I'm worried that at some point in time this help page will be longer than single page which makes it hard to read, see `kubectl run --help` as an example of what I don't want to have.